managed-services/transactional-email/troubleshooting/delist-domain.mdx

```diff
@@ -0,0 +1,32 @@
+---
+meta:
+  title: Getting delisted from Spamhaus
+  description: This page helps you delist your domain from Spamhaus' Block List
+content:
+  h1: Getting delisted from Spamhaus
+  paragraph: This page helps you delist your domain from Spamhaus' Block List
+tags: delist domain spam spamhaus
+dates:
+  validation: 2023-03-20
+  posted: 2023-03-20
+categories:
+  - managed-services
+---
+
+<Macro id="iam-requirements" />
+
+Spamhaus is a database that analyzes data and lists internet resources such as IP addresses or domains with poor [reputation](/managed-services/transactional-email/concepts/#ip-reputation). Spamhaus tracks malicious activity (ransomware, malware, phishing, and spam) and produces several block lists that mailbox providers use to protect themselves against malicious activity.
```

Comment:
There is twice "malicious" in one sentence. Maybe replace one of them with another word